src/app/content/practiceQuestions/components/PracticeQuestionsPopup.tsx

```diff
@@ -73,7 +74,8 @@ const PracticeQuestionsPopup = () => {
         </Header>
         <ShowPracticeQuestions />
       </Modal>
-    </PopupWrapper>
+    </PopupWrapper>,
+    assertDocument().body
```

Comment:
also i'm not sure you're supposed to portal directly into the body, i think you might be supposed to create a mount component and add it to the body and portal into that, but i'm not totally sure, if you saw this in the docs then its fine

Reply:
I've neverd heard of this and docs just says:
```
render() {
  // React does *not* create a new div. It renders the children into `domNode`.
  // `domNode` is any valid DOM node, regardless of its location in the DOM.
  return ReactDOM.createPortal(
    this.props.children,
    domNode  );
}
```

There is also closed issue on Material-UI repo according to this
https://github.com/mui-org/material-ui/issues/21626

Reply:
/shrug good enough for material-ui good enough for me